packages/wells/README.md

```diff
@@ -164,15 +164,55 @@ const retrievedCrs = wells.items.map(well => well.wellhead?.crs)
 
 ### _Filter wells by npt events_
 
+This allows you to filter wells on associated npt events.
+
+**duration**: how long did the event last.
+
+**measuredDepth**: at what depth the event occured.
+
+**nptCodes**: what npt codes are associated to the event.
+
+**nptCodeDetail**: what nptCodeDetails are associated to the event
```

Comment:
super nit: `npt codes` on line 173 and `nptCodeDetails` on line 175. In the description.